package.json

```diff
@@ -14,6 +14,7 @@
     "express-handlebars": "^3.0.0",
     "helmet": "^3.6.1",
     "jquery": "^3.2.1",
+    "jquery-circle-progress": "^1.2.2",
```

Comment:
AH! I think you added a dependency to package.json, but didn't update the package-lock.json shrinkwrap file, which _may_ explain why Circle-CI is blowing up.

It works for me locally if I do this:

```sh
$ rm -rf node_modules package-lock.json
$ npm i && npm run lint
```